### ISSUE TITLE
Queries on Data and Methods

### ISSUE THREAD

**Issue by patel-zeel** [AleksandarSekulic/MeteoSerbia1km#2]
Hi,

I am reproducing this work for a project of mine and I had several questions for which I could not get the answers from the paper. Can you please help me with those?

1. How missing data is handled during LLO CV? What if a station's information is not available on a particular day?
2. N nearest stations' information is directly used in the model or only via IDW? if used directly, can you please elaborate more on how exactly?

Thank you,

**Comment by AleksandarSekulic** [comment]
Hi,

Sorry for the late response, I was on a vacation.

1. We haven't handled missing data at all because it was not needed for the LLOCV. If an observation is not available on a particular day, it was not used in the LLOCV, i.e. we did not make a prediction for that missing observation.
2. N nearest stations were used directly in the RFSI model. I refer to the RFSI paper (https://www.mdpi.com/2072-4292/12/10/1687/htm) where the whole RFSI modelling process is explained. All the functionalities are implemented in the R package meteo (https://github.com/AleksandarSekulic/Rmeteo). Shortly, when we do modelling, we find e.g. 10 nearest observations and calculate distances to them for each observation location and for every day. This produces 20 new predictors (2 x N new predictors: 1st nearest observations, 2nd nearest observations, ... 10th nearest observations, distance to 1st nearest observations, distance to 2nd nearest observations, ... distance to 10th nearest observations). This way we create a regression matrix with dimensions: number of observations X (2 x N + P), where P is number of environmental predictors. When we want to make a prediction, we just find 10 nearest observation and distances to them from a prediction location (2 x N predictors) and again create a regression matrix with the same number of columns (2 x N + P).
Besides that, we also used IDW prediction in a specific location as a predictor (of course, we did not use the observation at a specific location for IDW prediction at that location).

Hope this helps. If you have any further questions, I'll be glad to help.
